drivers/builtin/include/mbedtls/pk.h

```diff
@@ -283,8 +283,6 @@ typedef int (*mbedtls_pk_rsa_alt_decrypt_func)(void *ctx, size_t *olen,
                                                const unsigned char *input, unsigned char *output,
                                                size_t output_max_len);
 typedef int (*mbedtls_pk_rsa_alt_sign_func)(void *ctx,
-                                            int (*f_rng)(void *, unsigned char *, size_t),
```

Comment:
[about the commit message] Minor: this is only about PK, we don't have any x509 in this repo.